Extension/src/LanguageServer/client.ts

```diff
@@ -620,6 +644,57 @@ class DefaultClient implements Client {
         this.model.tagParserStatus.Value = notificationBody.status;
     }
 
+    private updateInactiveRegions(params: InactiveRegionParams): void {
+        let renderOptions: vscode.DecorationRenderOptions = {
+            light: { color: "rgba(175,175,175,1.0)" },
+            dark: { color: "rgba(155,155,155,1.0)" }
+        };
+        let decoration: vscode.TextEditorDecorationType = vscode.window.createTextEditorDecorationType(renderOptions);
+
+        // Recycle the active text decorations when we receive a new set of inactive regions
+        let valuePair: DecorationRangesPair = this.inactiveRegionsDecorations.get(params.uri);
+        if (valuePair) {
+            // The language server will send notifications regardless of whether the ranges have changed. Unfortunately we must check for changes ourselves
```

Comment:
The "unfortunately" part of this comment is implying that there is a problem with the language server we are working around.  As we discussed earlier, we are making a conscious choice to do it this way and avoid a second copy of the cached results in the language server.  Perhaps this is unfortunate, but it is deliberate, and I don't think this comment helps us understand the code better or feel better about that decision, so my vote is to remove it. 😉

Reply:
Good point! I totally agree with that and it will be removed. :smile: